tests/certificate.py

```diff
@@ -16,6 +16,25 @@
 ...
 -----END CERTIFICATE-----"""
 
+CERT_CONTENT_WITH_EMPTY_LINE="""-----BEGIN CERTIFICATE-----
```

Comment:
Is this supposed to support multiple certificates per section? For some reason I thought it was only one per section. Which is fine, as long as whatever consumes the .pem file is ok with that.

Reply:
Yes, I thought so as well.
But it shows up that at this point it is very likely there will be just single a bundle of certs (well-known path and name) that should be managed by user (documented). So no tooling to add certs to a bundle. Therefore Anaconda should rather be able to pass such a bundle via kickstart. And can't think of any other sane solution then being able to dump multiple certs to a single file (for example merge/append in case of the same target -  that should the tooling do, not Anaconda). Also I think the kickstart itself / parser should rather not impose restrictions on the content at this stage of maturity of the feature (actually it is almost just a dump of string).

Reply:
Just to be clear. It will allow this section to be used multiple times but also multiple certificates in one section. Am I correct?